docs/README-agent-install.md

```diff
@@ -164,318 +153,303 @@ Virtual service guestshell+ detail
 
 ~~~
 
-To resize the guestshell filesystem, use the `guestshell resize rootfs` command. To resize the guestshell memory allocation, use the `guestshell resize memory` command. These commands can be executed even when the guestshell is not yet enabled. Note that the resize command does not take effect until after the guestshell container is (re)started with the `guestshell reboot` or `guestshell enable` command.
+<p>
+`guestshell resize rootfs` will resize the guestshell filesystem, `guestshell resize memory` resizes memory limits. The resize commands do not take effect until after the guestshell container is (re)started by the `guestshell reboot` or `guestshell enable` commands.
 
-**Example.** Guestshell is currently enabled. Resize guestshell filesystem to 400MB and memory to 300MB:
+**Example.** Guestshell is currently enabled. Resize the limits to 400MB disk and 300MB memory.
 
 ~~~
-n3k# guestshell resize rootfs ?
-  <158-600>  New root filesystem size (in MB)
-
 n3k# guestshell resize rootfs 400
-Note: Please disable/enable or reboot the Guest shell for root filesystem to be resized
 
 n3k# guestshell resize memory 300
-Note: Please disable/enable or reboot the Guest shell for system memory to be resized
 
 n3k# guestshell reboot
-Access to the guest shell will be temporarily disabled while it reboots.
 Are you sure you want to reboot the guest shell? (y/n) [n] y
 ~~~
 
-**Example.** Guestshell is currently disabled. Resize guestshell filesystem to 400MB and memory to 300MB:
-
-~~~
-n3k# guestshell resize rootfs 400
-Note: Root filesystem will be resized on Guest shell enable
-
-n3k# guestshell resize memory 300
-Note: System memory will be resized on Guest shell enable
 
-n3k# guestshell enable
-~~~
+#### *Step 2. Set Up Guestshell Network*
 
-See [References](#references) for more guestshell documentation.
+The `guestshell` is an independent CentOS container that does not inherit settings from NX-OS; thus it requires additional network configuration.
 
-#### Set Up Guestshell Network
+* Enter the guestshell environment using the `guestshell` command
+* Use `sudo` to become root. *Optional: Use `chvrf` to specify a vrf namespace; e.g. `sudo chvrf management`*
```

Comment:
We should use consistent verbage for both bash shell and guestshell when using the management vrf.